library/lib_exec_node.ks

```diff
@@ -0,0 +1,50 @@
+@lazyglobal off.
```

Comment:
Needs // This file is distributed under the terms of the MIT license, (c) the KSLib team